source_handlers_test.go

```diff
@@ -913,6 +916,19 @@ func TestSourceGet(t *testing.T) {
 	if *outSrc.Name != "Source1" {
 		t.Error("ghosts infected the return")
 	}
+
+	if outSrc.ID != fmt.Sprintf("%d", sourceId) {
+		t.Errorf("source with wrong ID returned, expected %d, got %s", sourceId, outSrc.ID)
+	}
+
+	for _, src := range fixtures.TestSourceData {
```

Comment:
I am not sure about this test. You search in `fixtures.TestSourceData`. Was intention to also  add` outSrc` somehow ?